core/src/main/java/org/elasticsearch/action/admin/indices/analyze/AnalyzeResponse.java

```diff
@@ -108,6 +117,12 @@ public void readFrom(StreamInput in) throws IOException {
             startOffset = in.readInt();
             endOffset = in.readInt();
             position = in.readVInt();
+            Integer len = in.readOptionalVInt();
```

Comment:
Do we need any versioning here?  Are we ever expected to be able to `readFrom` a node running an older ES version that doesn't send the `posLen`?